rust/Cargo.toml

```diff
@@ -157,9 +158,7 @@ polars = { version = "=0.50.0", features = [
     "string_normalize",
 ], optional = true }    
 polars-plan = { version = "=0.50.0", features = ["ffi_plugin", "python"], optional = true }
-polars-utils = { version = "=0.50.0" }
-# pinned due to: https://github.com/pola-rs/polars/issues/24208
-planus = { version = "=1.1.1" }
+polars-utils = { version = "=0.50.0", optional = true }
```

Comment:
Marking these as optional so that Polars crates aren't pulled in when building without Polars.